FF1Lib/Credits.cs

```diff
@@ -9,6 +9,15 @@ public partial class FF1Rom : NesRom
 		// A fun list of initial victory pages. Classic mistranslations.
 		private static readonly List<string[]> VictoryMessages = new List<string[]>
 		{
+			new [] // SMB
+			{
+				"",
+				"THANK YOU MARIO!",
+				"",
+				"BUT OUR PRINCESS",
+				"IS IN ANOTHER",
+				"CASTLE!",

```

Comment:
In keeping with what we do, this should be more like 
THANK YOU LIGHT WARRIORS
BUT OUR PRINCESS IS IN
ANOTHER TIMELINE

Reply:
I thought about that as well.  Message has been updated and should now look like this.

![image](https://user-images.githubusercontent.com/18019703/78951966-f3fd2b80-7aa1-11ea-97eb-8d31ad13011e.png)